Makefile

```diff
@@ -29,7 +29,7 @@ update-catalog-deployment:
 	curl 'https://api.mgmt.cloud.vmware.com/deployment/api/swagger/swagger/v2/api-docs?group=2019-01-15' | python3 -mjson.tool > swagger/vra-catalog-deployment.json
 
 update-iaas:
-	curl 'https://api.mgmt.cloud.vmware.com/iaas/api/swagger/swagger/v2/api-docs?group=iaas' | python3 -mjson.tool > swagger/vra-iaas.json
+	curl --insecure 'https://${swagger_endpoint}/iaas/api/swagger/swagger/v2/api-docs?group=iaas' | python3 -mjson.tool > swagger/vra-iaas.json
```

Comment:
Should define swagger_endpoint in this Makefile and then you can override when running make:
swagger_endpoint ?= api.mgmt.cloud.vmware.com

Also be good to make this same change on all of the curl calls.